kettle-plugins/hadoop-cluster/ui/src/main/javascript/app/components/newEdit/newEdit.component.js

```diff
@@ -155,6 +158,20 @@ define([
       vm.shimVersion = vm.data.model.shimVersion;
     }
 
+    // Prevent pressing the slash key, not allowed in cluster name
+    function onNameKeyDown($event) {
```

Comment:
From what I understand its not the angular way to inject DOM events into a controller function, but instead create a directive. Creating a directive is a lot of code for such a simple action, I'm good with this solution.